### DISCUSSION TITLE
How to configure column gutter sizes in typst?

### DISCUSSION THREAD

**Discussion by Canadauni** [quarto-dev/quarto-cli#8531] [Q&A]
Hello!

I've been really enjoying trying out quarto with typst today. One thing I ran up against is that when using the block layout notation ie `::: layout-ncol=4` is that the gutter sizes are quite a bit bigger than I would like. It looks like they are being set to 1em in the generated typst grid function. I'm wondering if it would be possible to remove the default 1em gutter so that it can be set in a raw typst block? ie `#set grid(gutter = 3pt)` or am I missing some other method to alter the gutter size?

Thanks!

**Comment by cscheid** [comment]
Not yet.
